release_notes/ocp-4-6-release-notes.adoc

```diff
@@ -2511,3 +2511,67 @@ link:https://access.redhat.com/solutions/5756401[{product-title} 4.6.15 containe
 ==== Upgrading
 
 To upgrade an existing {product-title} 4.6 cluster to this latest release, see xref:../updating/updating-cluster-cli.adoc#update-service-overview_updating-cluster-cli[Updating a cluster by using the CLI] for instructions.
+
+[id="ocp-4-6-16"]
+=== RHSA-2021:0308 - {product-title} 4.6.16 bug fix and security update
+
+Issued: 2021-02-08
+
+{product-title} release 4.6.16, which includes security updates, is now available. The list of bug fixes that are included in the update is documented in the link:https://access.redhat.com/errata/RHSA-2021:0308[RHSA-2021:0308] advisory. The RPM packages that are included in the update are provided by the link:https://access.redhat.com/errata/RHSA-2021:0309[RHSA-2021:0309] advisory.
+
+Space precluded documenting all of the container images for this release in the advisory. See the following article for notes on the container images in this release:
+
+link:https://access.redhat.com/solutions/5780541[{product-title} 4.6.16 container image list]
+
+[id="ocp-4-6-16-features"]
+==== Features
+
+[id="ocp-4-6-16-insights-operator-enhancements"]
+===== Enhancements to Insight Operator
```

Comment:
Should it be "Insights" like on the next line?

Reply:
lol yes thank you